methods/renderData.js

```diff
@@ -25,15 +26,16 @@ module.exports = function(payload) {
   const finalDetails = _.defaults({}, payload, emailDetails, defaultDetails);
 
   const renderedDetails = _.reduce(finalDetails, (result, value, key) => {
-    if (['to', 'template', 'data', 'text', 'headers'].indexOf(key) !== -1) {
+    if (['to', 'template', 'data', 'text', 'headers', 'inlineCss'].indexOf(key) > -1) {
       result[key] = value;
+      if (payload.inlineCss && (key === 'template' || key === 'text')) {
```

Comment:
Is there a possibility that the css inline functionality would fire before the template rendering?